packages/atomic/src/components/insight/atomic-insight-result-action/atomic-insight-result-action.tsx

```diff
@@ -56,12 +62,49 @@ export class AtomicInsightResultAction implements InitializableComponent {
    */
   @Prop({mutable: true}) public tooltip = '';
 
+  /**
+   * The text tooltip to show on the result action icon for some time after clicking the button.
+   */
+  @Prop({mutable: true}) public tooltipOnClick = '';
+
   /**
    * The type of action to perform when the result action is clicked. This will be sent along the event fired when the button is clicked.
    */
   @Prop({mutable: true}) public action: Actions | string = '';
 
+  private actions!: InsightAnalyticsActionCreators;
+
+  public initialize() {
+    this.actions = loadInsightAnalyticsActions(this.bindings.engine);
+  }
+
   private onClick() {
+    if (this.tooltipOnClick) {
+      const originalTooltip = this.tooltip;
+      this.tooltip = this.tooltipOnClick;
+      setTimeout(() => {
+        this.tooltip = originalTooltip;
+      }, 1000);
+    }
```

Comment:
Should we put an option to pass a custom TooltipOnClickDuration in order to override the one second timeout or one second shall be good most of the time ?